packages/calypso-e2e/src/data-helper.ts

```diff
@@ -1,8 +1,24 @@
 /**
  * External dependencies
  */
+import phrase from 'asana-phrase';
 import config from 'config';
 
+/**
+ *
+ */
+declare global {
+	interface String {
+		toProperCase(): string;
+	}
+}
+
+String.prototype.toProperCase = function (): string {
+	return this.replace( /\w\S*/g, function ( txt ) {
+		return txt.charAt( 0 ).toUpperCase() + txt.substr( 1 ).toLowerCase();
+	} );
+};
```

Comment:
What's the benefit of extending the prototype vs creating a `properCase` utility (lodash-style)?

Reply:
No particular reason - I took what was in the Seleneium-based `dataHelper`. [This StackOverflow](https://stackoverflow.com/questions/8392035/add-method-to-string-class) also was the first hit when I searched `add method to string class` so I assumed extending the prototype is an acceptable approach.

Would you like me to use the alternative approach you suggested (creating utility with lodash)?

Reply:
+1 to create a separate utility (and not using `lodash` for it, see p4TIVU-9Bf-p2).
Extending the prototype is not very common anymore in JS projects.

Reply:
Thanks for the context, @worldomonation. I think it would be good to refactor it to the functional `properCase` utility. It should be a quick one so I think we can still do it within this PR 👍

Reply:
Thanks both for the context. I'll push up a change in the next PR.